app/src/js/components/Operations/index.js

```diff
@@ -1,44 +1,43 @@
 import React from 'react';
-import { withRouter, Route } from 'react-router-dom';
+import { Route, Routes, Navigate } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
 import { Helmet } from 'react-helmet';
-import { connect } from 'react-redux';
-import withQueryParams from 'react-router-query-params';
 import PropTypes from 'prop-types';
 import Sidebar from '../Sidebar/sidebar';
 import DatePickerHeader from '../DatePickerHeader/DatePickerHeader';
 import OperationOverview from './overview';
 import { listOperations } from '../../actions';
 import { strings } from '../locale';
-import { filterQueryParams } from '../../utils/url-helper';
+import { withUrlHelper } from '../../withUrlHelper';
 
-const Operations = ({ dispatch, location, params, queryParams }) => {
-  const { pathname } = location;
+const Operations = ({ urlHelper }) => {
+  const dispatch = useDispatch();
+  const { queryParams, filterQueryParams, location, params } = urlHelper;
   const filteredQueryParams = filterQueryParams(queryParams);
+  const { pathname } = location;
 
   function query() {
     dispatch(listOperations(filteredQueryParams));
   }
 
   return (
-    <div className="page__workflows">
+    <div className='page__workflows'>
       <Helmet>
         <title> Cumulus Operations </title>
       </Helmet>
       <DatePickerHeader onChange={query} heading={strings.operations} />
-      <div className="page__content">
-        <div className="wrapper__sidebar">
+      <div className='page__content'>
+        <div className='wrapper__sidebar'>
           <Sidebar currentPath={pathname} params={params} />
-          <div className="page__content--shortened">
-            <Route
-              exact
-              path="/operations"
-              render={(props) => (
-                <OperationOverview
-                  {...props}
-                  queryParams={filteredQueryParams}
-                />
-              )}
-            />
+          <div className='page__content--shortened'>
+            <Routes>
+              <Route index element={<Navigate to="all" replace />} />
+              <Route
+                path='all'
```

Comment:
Is this catch-all because we only have /operations and and sub-routes should just render OperationOverview?

Reply:
You could say that. It's a best practice to have the full URL. It will show the complete path with the filter queried parameters so this is what will show in the URL if user searchs and filters for an ID with status of 'succeeded':/operations/all?e8e1e4ae-acee-43cb-bae6-bc3d95f98736=&limit=50&page=2&status=SUCCEEDED.